### PR TITLE
Theme refactoring

### PR DESCRIPTION
**Pull request by connyduck**
My goal for this is making the theming easier to understand and quicker to change in the future.

- use more defaults from the material design lib
- give attributes, colors and drawables more meaningful names
- remove custom theme attributes as much as possible
- instead of two themes for day/night, use one day&night theme for both
- merge the duplicate drawables we used to have & color them via attributes instead

### CODE REVIEW

File: app/src/main/res/drawable/avatar_border.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android">
-    <solid android:color="?attr/window_background" />
+    <solid android:color="?android:attr/windowBackground" />
```
Comment (charlag) [inline app/src/main/res/drawable/avatar_border.xml]: and attrs works here on api 21?

File: app/src/main/res/drawable/ic_clear_24dp.xml [hunk 2]
```diff
@@ -4,6 +4,6 @@
         android:viewportWidth="24.0"
         android:viewportHeight="24.0">
     <path
-        android:fillColor="@color/toolbar_icon_dark"
+        android:fillColor="#fff"
```
Comment (charlag) [inline app/src/main/res/drawable/ic_clear_24dp.xml]: `@android:color/white`?

Comment (charlag) [reply app/src/main/res/drawable/ic_clear_24dp.xml]: or `@color/white`

**Comment by connyduck** [comment]
I developed most of this on a Android 5 device, so I am sure it works there, but apparently I messed up the features for newer Android versions. Will fix, thx!

**Pull request merged**